src/Lykke.SettingsReader/ReloadingManager/SettingsConfiguratorExtensions.cs

```diff
@@ -37,36 +37,45 @@ public static class SettingsConfiguratorExtensions
 
             if (settingsUrl.StartsWith("http", StringComparison.InvariantCultureIgnoreCase))
             {
-                return new SettingsServiceReloadingManager<TSettings>(settingsUrl, configure);
+                return new SettingsServiceReloadingManager<TSettings>(settingsUrl, null, configure);
             }
 
-            return new LocalSettingsReloadingManager<TSettings>(settingsUrl);
+            return new LocalSettingsReloadingManager<TSettings>(settingsUrl, null);
         }
 
         /// <summary>
-        /// Checks dependencies
+        /// Loads settings
         /// </summary>
         /// <param name="configuration"></param>
-        /// <param name="settings">settings model to check dependencies</param>
-        /// <param name="slackConnString">slack connection string to send message about failed dependencies</param>
-        /// <param name="queueName">queue name for slack notifications</param>
-        /// <param name="sender">name of the sender in slack message</param>
-        /// <typeparam name="TSettings"></typeparam>
+        /// <param name="slackCheckerSettings"></param>
+        /// <param name="key">key name in the configuration</param>
+        /// <param name="configure">action to configure settings</param>
+        /// <typeparam name="TSettings">model for settings</typeparam>
         /// <returns></returns>
-        public static Task CheckDependenciesAsync<TSettings>(
-            this IConfiguration configuration, TSettings settings, string slackConnString, string queueName, string sender)
+        /// <exception cref="InvalidOperationException"></exception>
+        [Obsolete("Use LoadSettings method with Func")]
```

Comment:
This Obsolete is redundant

Reply:
no, the new version of LoadSettings should be used - to pass data for slack notification. This method won't send any slack notifications

Reply:
It IS the new version of LoadSettins. Or not?